scripts/release.bat

```diff
@@ -27,7 +27,19 @@ REM Copyright (c) 2016 solidity contributors.
 REM ---------------------------------------------------------------------------
 
 set CONFIGURATION=%1
+set VERSION=%2
+
+IF "%VERSION%"=="2015" (
+    set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvc*.dll"
+) ELSE (
+
+    IF EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\" (
+        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\msvc*.dll"
+		) ELSE (
+        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\msvc*.dll"
+    )
+)
 
 7z a solidity-windows.zip ^
```

Comment:
This will create the same file twice - won't this cause any problems?

Reply:
@chriseth I don't get it, why twice?

Reply:
There are two runs, they generate the same file which is uploaded to e.g. the github release page. It might make sense to just ignore one version, but then we should explicitly do that (probably the one with the older compiler), otherwise we do not know which one will end up on the release page.

Reply:
ok, i see, i'll update appveyor.yml to run release only on vs 2015. But i'll keep the change in this file, because in the future you may need to use 2017, and locally you could run this script with vs 2017.

what do you think?

Reply:
ok, great!

Reply:
@chriseth sorry to ask again, because I just updated my previous comment (my bad)
just to make sure, use 2017 to release, and those warning `Unknown compiler version - please run the configure tests and report `  could be ignored, right?